x-pack/plugin/shutdown/src/internalClusterTest/java/org/elasticsearch/xpack/shutdown/DesiredBalanceShutdownIT.java

```diff
@@ -89,7 +89,6 @@ public Settings onNodeStopped(String newNodeName) {
                     .stream()
                     .allMatch(s -> s.overallStatus() == SingleNodeShutdownMetadata.Status.COMPLETE)
             );
-        });
+        }, 120, TimeUnit.SECONDS);
```

Comment:
This may be unnecessary but 30s seems like it might be a _little_ short to relocate 5 shards if the CI worker is running egregiously slowly.

Reply:
NIT: should we make it dependent on the shard count?

Reply:
not really worth it IMO but I can if you insist :)